drivers/include/periph/pwm.h

```diff
@@ -116,40 +156,25 @@ uint8_t pwm_channels(pwm_t dev);
 void pwm_set(pwm_t dev, uint8_t channel, uint16_t value);
 
 /**
- * @brief   Start PWM generation on the given device
- *
- * @param[in] dev           device to start
- */
-void pwm_start(pwm_t dev);
-
-/**
- * @brief   Stop PWM generation on the given device
+ * @brief   Resume PWM generation on the given device
  *
- * @param[in] dev           device to stop
- */
-void pwm_stop(pwm_t dev);
-
-/**
- * @brief   Power on the PWM device
+ * When this function is called, the given PWM device is powered on and
+ * continues its previously configured operation for each channel, where each
```

Comment:
'where each' looks strange, what about 'each of them' (I guess it's the channels) ?